.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
```

Comment:
@jhryniuk please remove `5.*` version, keep only `7.0` and add `7.1` with allowed failures